eclipse-collections/src/main/java/org/eclipse/collections/impl/lazy/parallel/list/ListIterableParallelIterable.java

```diff
@@ -85,12 +85,6 @@ public T detect(Predicate<? super T> predicate)
     }
 
     @Override
-    public <V> ParallelListIterable<V> flatCollect(Function<? super T, ? extends Iterable<V>> function)
-    {
-        return new ParallelFlatCollectListIterable<T, V>(this, function);
-    }
-
```

Comment:
Sorry, please ignore above. I was looking at the incorrect class. 

It's not changing the behaviour so this is ok.